scripts/base/frameworks/analyzer/main.zeek

```diff
@@ -163,6 +172,23 @@ export {
 	##
 	## This set can be added to via :zeek:see:`redef`.
 	global requested_analyzers: set[AllAnalyzers::Tag] = {} &redef;
+
+	## Event that is raised when an analyzer raised a service violation and was
+	## removed.
```

Comment:
The extra event definitely helps understand the control flow!

I have a nit/naming thoughts: `analyzer_violation_info()` and `analyzer_failed()` are pretty close, but the "analyzer was removed" is a significant difference between them. So was pondering proposing `analyzer_removed(...)`. But that's not great due to removals happening for benign reasons (`SSH`, `SSL`). 

Maybe you have another idea, otherwise feel free to resolve, anything I have in mind right now ends-up being a mouthful - e.g `violating_analyzer_removed()` :-(

Reply:
yeah, I had problems coming up with a name that I liked.

`analyzer_removed` seems more confusing to me than the current state. If anyone else has a nice idea for a name - please let me know.